common/lib/populate-resources.js

```diff
@@ -1,16 +1,14 @@
-const moveService = require('../services/move')
 const personEscortRecordService = require('../services/person-escort-record')
 const referenceDataService = require('../services/reference-data')
 
 const findUnpopulatedResources = require('./find-unpopulated-resources')
 
-const lookupMethods = {
-  locations: referenceDataService.getLocationById,
-  moves: moveService.getById,
-  person_escort_records: personEscortRecordService.getById,
-}
-
-const populateResources = async (obj, options, processed = []) => {
+const populateResources = async (obj, req, options, processed = []) => {
```

Comment:
Changes to this are in a separate commit if you'd like to see clearly what's changed here. This uses a few services, the move one is the first to touch it. It needs the req object passing so that the service calls it makes are correctly tracked. It's only used in one place https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/1104/files#diff-6c64412d97734c6e716b2d59530cfccb4a6b938b2897fb738f67d7ed01f1b0fdR24